### PR TITLE
Issue#2 - Code Stat Model

### PR DESCRIPTION
**Pull request by jc-hayes**
Created the GameStat Struct Model & Unit Tests for GameStatStruct

### CODE REVIEW

**Review by Juanitaag1 — APPROVED** [review]
looks good James.

**Review by MeyAyre — APPROVED** [review]
looks good

File: GYBITG/Models/GameStat.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+//
+
+import Foundation
+
+struct GameStat {
+    
+    let statId: Int
+    let userId: String
+    let gameDate: Date
+    let points: Int?
```
Comment (Bansenauer-Cascadia) [inline GYBITG/Models/GameStat.swift:15]: You can assign a default value in the definition

File: GYBITG/Models/GameStat.swift [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    let rebounds: Int?
+    let assists: Int?
+    let steals: Int?
+    let blocks: Int?
+    let minutesPlayed: Double?
+
+    init(userId: String, statId: Int, gameDate: Date, points: Int? = 0, rebounds: Int? = 0, assists: Int? = 0,steals: Int? = 0, blocks: Int? = 0, minutesPlayed: Double? = 0.0) {
```
Comment (Bansenauer-Cascadia) [inline GYBITG/Models/GameStat.swift:22]: This is better though

**Review by Bansenauer-Cascadia — APPROVED** [review]
Good - follow up with Bobby about whether all these apply in every game (optionals or not)

**Pull request merged**